library/src/lib/calendar/calendar.component.ts

```diff
@@ -614,7 +614,23 @@ export class CalendarComponent implements OnInit, OnDestroy, AfterViewChecked, C
     }
 
     setCurrentMonth(month: number) {
+        // get the current date of the month
+        const currentDate = this.date.getDate();
+        // get the number of days in the new month
+        const daysInNewMonth = new Date(this.date.getFullYear(), month + 1, 0).getDate();
```

Comment:
if this is december, will the Date object automatically go to January of next year?

Reply:
yes
```
new Date(2019, 12, 1) = Wed Jan 01 2020 00:00:00 GMT-0700 (Mountain Standard Time)
```